.github/workflows/ci.yml

```
@@ -91,7 +91,7 @@ jobs:
           MONGO_URL: "mongodb://localhost:27017"
           DB_NAME: "llm_wiki_dashboard_ci"
         run: |
-          pytest -x -v --tb=short --timeout=120
+    pytest -x -v --tb=short --timeout=120 --ignore=tests/test_hardware.py --ignore=tests/test_backend_runtime_bootstrap.py
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)